### DISCUSSION TITLE
Not possible to get Github Copilot from GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by s0539722** [community/community#130133] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,
I recently was approved for GitHub Student Developer Pack and I'm able to log in to https://education.github.com/
It states I can now get Copilot from within my account settings in https://github.com/settings/copilot.

Unfortunately I can't choose Copilot there. It just tells me I should buy it now.
![Screenshot 2024-06-28 090404](https://github.com/community/community/assets/9092775/9305be2f-9a46-448f-8344-df2bde0e768f)

**Comment by jetocotoje** [comment, marked as answer]
For some reason, you need to enable it on your settings page (https://github.com/settings/billing/summary). I had the same issue.

**Reply by s0539722** [reply]
Thank you so much!
